client/indexer-db/src/models/proofs.rs

```diff
@@ -0,0 +1,55 @@
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+
+use crate::{schema::proofs, DbConnection};
+
+/// A single record table that holds the state/metadata of the indexer service.
+#[derive(Debug, Queryable, Insertable, Selectable)]
+#[diesel(table_name = proofs)]
+pub struct Proofs {
+    pub id: i32,
+    pub provider_id: String,
+    pub last_tick_proof: i64,
+}
+
+impl Proofs {
+    pub async fn create<'a>(
+        conn: &mut DbConnection<'a>,
+        provider_id: String,
+        last_tick_proof: i64,
```

Comment:
I'd rename this to `last_tick_proven`, or else have two fields stored here: `last_tick_proven` and `proof_submitted_at_tick` (or similar name). The former would be the last tick for which this `provider` submitted a proof, and the latter would be the tick at which that proof was submitted.